package.json

```diff
@@ -1207,6 +1207,7 @@
     "io-ts": "^2.2.22",
     "ipaddr.js": "2.0.0",
     "isbinaryfile": "4.0.2",
+    "jest-diff": "^29.7.0",
```

Comment:
Any idea what triggered this reordering? I'd perhaps revert this change because the version is the same.

Reply:
Ah, got it! The dangers of only looking at the diff haha.